src/LaunchDarkly/ApcuLDDFeatureRequester.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
I've merged #55 so that should be be ok now. Yes please update the docs regarding apc vs. apcu. Your changes are definitely welcome.
Thanks!